connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -429,4 +489,69 @@ private void deleteAllTopics(EmbeddedKafkaCluster cluster) {
         } catch (Throwable e) {
         }
     }
+
+    private void produceMessages(EmbeddedConnectCluster cluster, String topicName, String msgPrefix) {
+        produceMessages(cluster, topicName, msgPrefix, null);
```

Comment:
I wonder if we should pass `NUM_PARTITIONS` instead of `null` for the last argument. Then `numPartitions` can be an `int` in the other `produceMessages()` method. WDYT?